aws/coreos-docker-servers.tf

```diff
@@ -30,3 +33,23 @@ resource "template_file" "etcd_discovery_url" {
     }
   }
 }
+
+resource "aws_security_group" "docker_node" {
```

Comment:
I think we should block it. Even though the implementation is kind of ugly and it doesn't cover container -> daemon.